koku/reporting/migrations/0213_aws_rm_unused_partables.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.1.13 on 2022-01-05 20:47
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [("reporting", "0212_auto_20211203_1640")]
+
+    operations = [
+        migrations.RemoveField(model_name="awscomputesummarybyservicep", name="account_alias"),
+        migrations.RemoveField(model_name="awscomputesummarybyservicep", name="organizational_unit"),
+        migrations.RemoveField(model_name="awscomputesummarybyservicep", name="source_uuid"),
+        migrations.RemoveField(model_name="awsstoragesummarybyregionp", name="account_alias"),
+        migrations.RemoveField(model_name="awsstoragesummarybyregionp", name="organizational_unit"),
+        migrations.RemoveField(model_name="awsstoragesummarybyregionp", name="source_uuid"),
+        migrations.RemoveField(model_name="awsstoragesummarybyservicep", name="account_alias"),
+        migrations.RemoveField(model_name="awsstoragesummarybyservicep", name="organizational_unit"),
+        migrations.RemoveField(model_name="awsstoragesummarybyservicep", name="source_uuid"),
```

Comment:
Why RemoveField when you're removing the model? Am I missing something?

Reply:
To be honest I autogenerated this migration. I thought it was weird to drop those fields first too, but then I noticed that all of these fields were foreign keys. I think this is Django's way to handle potential cascading deletes on the foreign keys values before the model is deleted. That or to check if there are any foreign key constraints that would prevent the models from deleting.